test/tools/ossfuzz/abiV2FuzzerCommon.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <libsolidity/interface/CompilerStack.h>
+#include <libyul/AssemblyStack.h>
+#include <liblangutil/Exceptions.h>
+#include <liblangutil/SourceReferenceFormatter.h>
+#include <libdevcore/Keccak256.h>
+
+namespace dev
+{
+namespace test
+{
+namespace abiv2fuzzer
+{
+class SolidityCompilationFramework
+{
+public:
+	explicit SolidityCompilationFramework(langutil::EVMVersion _evmVersion = {});
+
+	Json::Value getMethodIdentifiers()
+	{
+		return m_compiler.methodIdentifiers(m_compiler.lastContractName());
+	}
+	dev::bytes compileContract(std::string const& _sourceCode, std::string const& _contractName = {});
```

Comment:
Compile the contract returned by a call to `m_compiler.lastContractName()` if `_contractName` is empty.